packages/core-modules/contracts/modules/ElectionModule.sol

```diff
@@ -28,13 +28,10 @@ contract ElectionModule is
         uint8 minimumActiveMembers,
         uint64 nominationPeriodStartDate,
         uint64 votingPeriodStartDate,
-        uint64 epochEndDate,
-        address debtShareContract
-    ) external override onlyOwner onlyIfNotInitialized {
+        uint64 epochEndDate
+    ) public override onlyOwner onlyIfNotInitialized {
```

Comment:
This is now public because the SNX specific ElectionModule will _overload_ this function and call it internally.